lake/data/seekindex.go

```diff
@@ -17,28 +21,37 @@ func LookupSeekRange(ctx context.Context, engine storage.Engine, path *storage.U
 		return nil, err
 	}
 	defer r.Close()
-	rg := &seekindex.Range{Start: -1}
-	reader := seekindex.NewSectionReader(r, obj.Last, obj.Count, obj.Size, cmp)
+	var rg *seekindex.Range
+	unmarshaler := zson.NewZNGUnmarshaler()
+	reader := zngio.NewReader(zed.NewContext(), r)
 	swapper := expr.NewValueCompareFn(order.Asc, o == order.Asc)
 	for {
-		s, err := reader.Next()
-		if s == nil || err != nil {
+		val, err := reader.Read()
+		if val == nil || err != nil {
 			return rg, err
 		}
-		first := s.Keys.First()
-		last := s.Keys.Last()
-		if swapper(first, last) > 0 {
-			first, last = last, first
+		var entry seekindex.Entry
+		if err := unmarshaler.Unmarshal(val, &entry); err != nil {
+			return nil, fmt.Errorf("corrupt seek index entry for %q at value: %q (%w)", obj.ID.String(), zson.String(val), err)
 		}
-		if filter != nil && filter.Eval(first, last) {
-			continue
+		from := entry.From
+		to := entry.To
+		if swapper(from, to) > 0 {
+			from, to = to, from
 		}
-		if countSpan != nil && !countSpan.Overlaps(s.Counts.First(), s.Counts.Last()) {
+		if filter != nil && filter.Eval(from, to) {
 			continue
 		}
-		if rg.Start == -1 {
-			rg.Start = s.Range.Start
+		if countSpan != nil {
```

Comment:
If we're dropping indexes we no longer need this count stuff, but this can be handled later...

Reply:
Yes, this in my next PR.  I just wanted all the tests to continue to pass in this commit.